### ISSUE TITLE
Add a command to list progress breakdown for achievements

### ISSUE THREAD

**Issue by Mohron** [BadgeUp/sponge-client#15]
`/achievement` or a similar command should have a way to output a paginated list of achievements. Also you should be able to click on each achievement to get a list of the criteria associated with the achievement.

**Comment by Mohron** [comment]
Really need a way to see achievements and progress towards each criteria (can't even check this on the dashboard that I'm aware of). It's really hard to tell what exactly is going on without easy access to this information.

**Comment by MarkHerhold** [comment]
@Mohron Do you need a way to see progress for a single subject, or for an entire group of people? 

For instance, if you have a lot of users, would you like to see a distribution of achievement completeness across your population of users? Or is the need only to look at a single user's progress?

**Comment by Mohron** [comment]
@MarkHerhold A single subject like you've done would the Metrics would be very useful at this point. The overall stats is really not as important while testing even though it would be a nice feature to have.

**Comment by MarkHerhold** [comment]
This has been resolved, we'll continue working on the styling for more crowded charts like the one below. Try it out and let me know what you think!

![achievement-progress-chart](https://cloud.githubusercontent.com/assets/4934732/25060833/adc7ca14-2174-11e7-8035-9ec97822ddbb.png)

**Comment by Mohron** [comment]
The web dashboard feature is great! I hope a simpler version of this will still be made available in the plugin so a player can track their own progress.

**Comment by RobertHerhold** [comment]
@Mohron Yes, we closed this by accident. We will still be implementing an in-game feature!

**Comment by RobertHerhold** [comment]
This is mostly resolved by 83aa26b54015f01941945bfa33a90fac7f853eba, minus listing achievement criteria.

Preview:

![image](https://cloud.githubusercontent.com/assets/8069753/25105865/9825a970-2394-11e7-989b-7d3b8b753024.png)

Note that you will also need the `progress:read` scope in your API key as well

**Comment by Mohron** [comment]
Is this still underway? Also what testing are you needing to continue development? Things have been pretty quiet here.

**Comment by MarkHerhold** [comment]
This is hard to represent in MC because we would have to draw the entire eval tree in chat. This could be both hard to implement and a little unwieldy to users.

I think the best option is to just describe how to earn the achievement in the description. Would that work for you @Mohron?
